### DISCUSSION TITLE
disable moving away of dock objects??

### DISCUSSION THREAD

**Discussion by davoodkharmanzar** [microsoft/MixedRealityToolkit-Unity#10246] [Q&A]
hi,
based on this [doc](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/experimental/dock?view=mrtkunity-2021-05) that mentioned **"_Docked objects automatically move away to make space for new objects..._"**
is it possible to disable this feature?
i need fix dock for specific object

**Comment by david-c-kline** [comment]
@cre8ivepark ?
